db/migrate/20230511192831_create_topics.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTopics < ActiveRecord::Migration[7.0]
+  def change
+    create_table :topics do |t|
+      t.string :name
+      t.string :image
+
+      t.timestamps
```

Comment:
By doing this you are adding this constraint at db level.

```suggestion
      t.string :name, null: false
      t.string :image

      t.timestamps, null: false
```

Reply:
Thanks for the review, its been changed